Include/README.rst

```diff
@@ -0,0 +1,68 @@
+The Python C API
+================
+
+The C API is divided into three sections:
+
+1. ``Include/``
+2. ``Include/cpython/``
+3. ``Include/internal/``
+
+
+Include: Limited API and Stable API
+===================================
+
+``Include/`` contains the public Limited API and the Stable API.  The
+Stable API is defined by :pep:`384`.
+
+Functions stealing references or returning borrowed references *must not*
+be added to the Limited API API or the Stable API.  A strong reference
+*must* be returned.  The API in ``Include/`` should not expose or leak
+implementation details.
+
+Please start a public discussion before adding new C API to ``Include/``.
+
+Functions or macros with a ``_Py`` prefix do not belong in ``Include/``.
+
+
+Include/cpython: CPython implementation details
+===============================================
+
+``Include/cpython/`` contains the public API that is excluded from the
+Limited and Stable API.  :pep:`384` introduced the ``Py_LIMITED_API``
+define to exclude functions from the Limited API.  When a new API is
+introduced, it has to be explicitly excluded using
+``#ifndef Py_LIMITED_API``.  If the author forgets it, the function is
+added to be Stable and Limited API by mistake!
```

Comment:
It doesn't work like that. Include/cpython/ header files are not included if Py_LIMITED_API macro is defined.

Reply:
C&P from https://bugs.python.org/issue35134#msg329060. I removed the last sentence. Let me know if you want to rephrase it.